### ISSUE TITLE
[@cypress/eslint-plugin-json]: Skipping file due to "json/json-with-comments-files" setting

### ISSUE THREAD

**Issue by SDAdham** [Bkucera/eslint-plugin-json-format#52]
Hello, I keep getting the following warning on my code-workspace and most of my tsconfig.json files

```
[@cypress/eslint-plugin-json]: Skipping file due to "json/json-with-comments-files" setting: 
  ["*.code-workspace", "**/tsconfig.json", ".vscode/**"]
To remove this warning add "MCZ-BeM.code-workspace" to your `.eslintignore` file
```

Can you please help?
